src/cdomain/value/cdomains/intDomain0.ml

```diff
@@ -69,7 +70,7 @@ let should_wrap ik = not (Cil.isSigned ik) || get_string "sem.int.signed_overflo
   * Always false for unsigned types, true for signed types if 'sem.int.signed_overflow' is 'assume_none'  *)
 let should_ignore_overflow ik = Cil.isSigned ik && get_string "sem.int.signed_overflow" = "assume_none"
 
-type overflow_info = { overflow: bool; underflow: bool;}
+type overflow_info = IntDomain_intf.overflow_info = { overflow: bool; underflow: bool;}
```

Comment:
Is this strictly necessary? Isn't `IntDomain_intf.overflow_info` a non-abstract type?